Create ui-c8y-1023-13-0-added-auto-or-dropdown-option-for-data-point-graph-legend-graft.md (#4002)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1023-13-0-added-auto-or-dropdown-option-for-data-point-graph-legend-graft.md

---
date: ""

> title: Add auto or dropdown option for data point graph legend

Reviewer: ```suggestion
title: Added option to control the data point graph legend
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-65466
version: 1023.13.0
---

> To improve the usability of data point graphs, a new option has been added to control the display of the graph legend. Users can now select between an automatic mode, which shows the legend if there is enough space, or explicitly control the legend visibility through a dropdown menu. This change enhances the flexibility of the graph configuration and allows users to optimize the visual representation based on their specific requirements and available screen space.

Reviewer: ```suggestion
To improve the usability of data point graphs, a new option has been added that allows users to control the display of the graph legend. They can now select between an automatic mode, which shows the legend if there is sufficient space, or explicitly control the legend visibility through a dropdown menu. This change enhances the flexibility of the graph configuration and allows users to optimize the visual representation based on their specific requirements and available screen space.
```
